Source/ConversationV1/Conversation.swift

```diff
@@ -72,9 +72,7 @@ public class Conversation {
         do {
             let json = try JSONWrapper(data: data)
             let code = response?.statusCode ?? 400
-            let message = try json.getString(at: "error")
-            let userInfo = [NSLocalizedDescriptionKey: message]
-            return NSError(domain: domain, code: code, userInfo: userInfo)
+            return NSError(domain: domain, code: code, userInfo: nil)
```

Comment:
These changes are related to error-handling. Right now, we require a person to identify the error fields (either by handwriting these couple of lines or passing information as a command-line argument to the generator).

This should improve soon, allowing us to generate and parse an error model.